src/lib/block_producer/block_producer.ml

```diff
@@ -71,6 +71,8 @@ let lift_sync f =
            [%log' error (Logger.create ())] "Ivar.fill bug is here!" ;
          Ivar.fill ivar (f ()) ) )
 
+let zkapp_cmd_limit = ref None
+
```

Comment:
@nholland94 Were you thinking somewhere around here like this? Thanks.

Reply:
Yes, except it should be initialized to the value from the compile time configuration.